theano/gpuarray/dnn.py

```diff
@@ -129,13 +136,12 @@ def _dnn_check_compile():
 def _dnn_check_version():
     v = version()
     if v < 5000:
-        return False, "cuDNN version is too old. Update to v5, was %d." % v
-    # 5200 should not print warning with cudnn 5.1 final.
+        return False, "cuDNN version is too old. Update to v5* or higher, was %d." % v
     if v >= 6100:
         warnings.warn("Your cuDNN version is more recent than "
                       "Theano. If you encounter problems, try "
                       "updating Theano or downgrading cuDNN to "
-                      "version 6.0.")
+                      "a version >= v5 and < v6.1.")
```

Comment:
I would use <= v6 as the upper limit since we don't know if there will be a v6.1 yet.